### ISSUE TITLE
Easier Access To Task Output

### ISSUE THREAD

**Issue by iswix-llc** [concourse/concourse#1400]
# Feature Request

## What challenge are you facing?

My tasks sometimes have a lot of text in them.  Say the console output of msbuild.exe.  When I see a failed built it is difficult to isolate the logs that I need to look through.

## A Modest Proposal

It would be nice if there was a "raw" button,  "download" button or even a "filter"  button that allowed me faster access / searching of the console output to look for my problem.

**Comment by chendrix** [comment]
Have you ever used `fly watch`?

**Comment by iswix-llc** [comment]
Yes but I was looking for a way to do this in the webui.  On the other hand I used to use a system called Build Forge where we had 800+ projects on each of two servers (because we found that was all one server coudl handle)  and ran over 100,000 builds a year.  We eventually stopped piping any text to the system and instead redirected it all to logfiles because we found BuildForge just couldn't manage that amount of data being loaded into it.  I only have 16 pipelines so far and I have no idea  how Concourse, PostGreSQL and our hardware will handle this.  But since the text is there in the web UI now I was hoping to be able to isolate and search it quickly without having to drop to a command prompt.
